### DISCUSSION TITLE
Unable to connect after a period of screen shutdown on the Android device

### DISCUSSION THREAD

**Discussion by HCreater** [rustdesk/rustdesk#9449] [Q&A]
The Android app was able to connect normally from the off screen state yesterday, but it doesn't work today. I have tried several versions, but after a period of off screen, I couldn't reconnect, but it kept connecting even after off screen. Moreover, it appeared on several Android phones at the same time. I feel like a modification in the official strategy of Rustdesk. How can I configure it to restore the ability to reconnect after off screen?

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/5627#discussioncomment-9680137
